update/common/scripts/5.4/updatepasswordhashlength.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
apart from @emodric comment, i'm a bit confused. why this is script is in ```/5.4``` but the sql is in ```/5.9```?